src/sentry/signals.py

```diff
@@ -72,22 +78,22 @@ def wrapped(func):
             wrapped.__doc__ = receiver.__doc__
         return wrapped(receiver)
 
-    def send_robust(self, sender, **named):
+    def send_robust(self, sender, **named) -> List[Tuple[Receiver, Union[Exception, Any]]]:
         """
         A reimplementation of send_robust which logs failures, thus recovering stacktraces.
         """
-        responses = []
+        responses: List[Tuple[Receiver, Union[Exception, Any]]] = []
         if not self.receivers or self.sender_receivers_cache.get(sender) is NO_RECEIVERS:
             return responses
 
         # Call each receiver with whatever arguments it can accept.
         # Return a list of tuple pairs [(receiver, response), ... ].
-        for receiver in self._live_receivers(sender):
+        for receiver in self._live_receivers(sender):  # type: ignore[attr-defined]
             try:
                 response = receiver(signal=self, sender=sender, **named)
             except Exception as err:
                 if "pytest" in sys.modules:
-                    if _receivers_that_raise is _all or receiver in _receivers_that_raise:
+                    if _receivers_that_raise is _AllReceivers or receiver in _receivers_that_raise:  # type: ignore[operator]
```

Comment:
mypy is just being dumb because it thinks that `receiver in` could run against `Union[Type[...]]`. IOW, `_receivers_that_raise is _AllReceivers` is just not a well-supported pattern

Reply:
i'll try the enum thing

Reply:
fixed with enum -- though i'm still not really using it as an enum, i'm just fixing this by using `isinstance`